assets/src/functions/hydrateBlock.js

```diff
@@ -1,4 +1,4 @@
-import {hydrate} from 'react-dom';
+import {hydrateRoot as hydrate} from 'react-dom';
```

Comment:
I also see this warning on local on pages with a CarouselHeader: 

<img width="357" alt="Screenshot 2023-05-05 at 15 28 46" src="https://user-images.githubusercontent.com/6949075/236470715-69987460-1342-4f82-a69c-f5ab7b3edd29.png">

And it's mentioned in the link you shared too

Reply:
I think this part should be fixed within [PLANET-6917](https://jira.greenpeace.org/browse/PLANET-6917), which I created too months ago.

Reply:
I think it would make sense to tackle here in this PR, since it seems it would go well with [this commit](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1042/commits/b020eae2859d2dad5522eb82bfd67e81a9ab41b1) (`Follow up React 18`)
I don't think it's realistic to remove all of our uses of `frontendRendered` in time for the update 🤔

Reply:
Update: created a follow-up [ticket](https://jira.greenpeace.org/browse/PLANET-7189) for this issue